e2e/tests/localAuthorityMaintainsCaseAfterSDO_test.js

```diff
@@ -0,0 +1,61 @@
+const config = require('../config.js');
+const hearingDetails = require('../fixtures/hearingTypeDetails.js');
+const directions = require('../fixtures/directions.js');
+
+let caseId;
+
+Feature('draft CMO after sdo');
+
+Before(async (I, caseViewPage, submitApplicationEventPage, sendCaseToGatekeeperEventPage, addHearingBookingDetailsEventPage, draftStandardDirectionsEventPage) => {
+  if (!caseId) {
+    await I.logInAndCreateCase(config.swanseaLocalAuthorityEmailUserOne, config.localAuthorityPassword);
+    await I.enterMandatoryFields();
+    await caseViewPage.goToNewActions(config.applicationActions.submitCase);
+    submitApplicationEventPage.giveConsent();
+    await I.completeEvent('Submit');
+
+    // eslint-disable-next-line require-atomic-updates
+    caseId = await I.grabTextFrom('.heading-h1');
+    console.log(`Case ${caseId} has been submitted`);
+
+    I.signOut();
+
+    //hmcts login and send to gatekeeper
```

Comment:
Hmm, maybe not one for this story but we do this elsewhere! It would be nice to have reusable methods for such tasks.